document/zh/deploy_radondb-clickhouse_operator_on_kubernetes.md

```diff
@@ -0,0 +1,170 @@
+Contents
+=================
+
+- [在 Kubernetes 上部署 RadonDB ClickHouse](#在-kubernetes-上部署-radondb-clickhouse)
+  - [简介](#简介)
+  - [部署准备](#部署准备)
+  - [部署步骤](#部署步骤)
+    - [步骤 1 : 添加仓库](#步骤-1--添加仓库)
+    - [步骤 2 : 部署 RadonDB ClickHouse Operator](#步骤-2--部署-radondb-clickhouse-operator)
+    - [步骤 3 : 部署 RadonDB ClickHouse 集群](#步骤-3--部署-radondb-clickhouse-集群)
+    - [步骤 4 : 部署校验](#步骤-4--部署校验)
+      - [查看 Pod 运行状态](#查看-pod-运行状态)
+      - [查询 SVC 运行状态](#查询-svc-运行状态)
+  - [访问 RadonDB ClickHouse](#访问-radondb-clickhouse)
+    - [通过 Pod](#通过-pod)
+    - [通过 Service](#通过-service)
+  - [持久化](#持久化)
+
+# 在 Kubernetes 上部署 RadonDB ClickHouse
+
+## 简介
+
+RadonDB ClickHouse 是基于 [ClickHouse](https://clickhouse.tech/) 的开源、高可用、云原生集群解决方案。
+
+本教程演示如何使用命令行在 Kubernetes 上部署 RadonDB ClickHouse。
+
+## 部署准备
+
+- 已成功部署 Kubernetes 集群。
+- 已安装 Helm 包管理工具。
+
+## 部署步骤
+
+### 步骤 1 : 添加仓库
+
+添加并更新 Helm 仓库。
+
+```bash
+$ helm repo add ck https://radondb.github.io/radondb-clickhouse-kubernetes/
+$ helm repo update
+```
+
+### 步骤 2 : 部署 RadonDB ClickHouse Operator
+
+```bash
+$ helm install clickhouse-operator ck/clickhouse-operator
```

Comment:
Should we tell users the following?
1. `clickhouse-operator` is the name of the application. Can be customized or automatically generated using `--generate-name`;
2. The namespace of the application deployment can be specified by `-n`;
3. Operator is best installed in the `kube-system` namespace, so that all namespaces can be monitored, otherwise only the namespace where it is located will be monitored

Reply:
1&2. add the command model
3. add the notice